code/game/objects/items/cards_ids.dm

```diff
@@ -168,11 +168,11 @@
 		if(!registered_account)
 			to_chat(usr, "There's no account registered!")
 			return
-		var/target_value = input(usr, "How much mining points do you want to give as? (negative value works)", "Give mining points", 100000) as num
+		var/target_value = input(usr, "How many mining points do you like to give? (use nagative to take)", "Give mining points") as num
```

Comment:
"How many mining points would you like to add? (use negative to take)"

Reply:
ahh fuck I forgot to edit it lmao